drivers/adc/Kconfig.mcux

```diff
@@ -48,3 +54,26 @@ config ADC_MCUX_ADC16_VREF_ALTERNATE
 endchoice
 
 endif # ADC_MCUX_ADC16
+
+
+if ADC_MCUX_LPADC
+
+config LPADC_DO_OFFSET_CALIBRATION
+	bool "Do offset calibration"
+	default n
+	help
+	  Do offset calibration
+
+config LPADC_OFFSET_VALUE_A
+	int "Offset value A"
+	default 10
+	help
+	  Offset value A to use if CONFIG_LPADC_DO_OFFSET_CALIBRATION is true
+
+config LPADC_OFFSET_VALUE_B
+	int "Offset value B"
+	default 10
+	help
+	  Offset value B to use if CONFIG_LPADC_DO_OFFSET_CALIBRATION is true
```

Comment:
I think these might be better as dts properties

Reply:
Changed to using dts properties.